src/views/trips/SingleRequestView.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import CommonTable from './CommonTable';
 
-const RequestsView = ({ data }) => <CommonTable trips={data && data.trips} comments={data && data.comments && data.comments.data || []} />;
+const RequestsView = ({ data, approveRequest, params }) => <CommonTable trips={data && data.trips} params={params} approveRequest={approveRequest} comments={data && data.comments && data.comments.data || []} />;
```

Comment:
@nziokaivy  can we implement the functionality for toggling between `approve` and `reject` buttons. By this , I mean if a manager clicks on approve, `approve` button gets disabled?
<img width="1440" alt="Screen Shot 2020-01-23 at 14 04 06" src="https://user-images.githubusercontent.com/35301129/72983420-e791fc80-3de9-11ea-8660-13b98265ffee.png">